crypto/des/fcrypt.c

```diff
@@ -115,17 +119,14 @@ char *DES_fcrypt(const char *buf, const char *salt, char *ret)
      * libraries.  People found that the disabled accounts effectively had no
      * passwd :-(.
      */
```

Comment:
I'm wondering what to do with this comment now that we require the salt to be at least 2 bytes long.